### PR TITLE
Add getToken function support to Luigi Container

### LINKED ISSUE #3784: Add client getToken function support to Container

### PR DESCRIPTION
**Pull request by ndricimrr**
**Description**

Changes proposed in this pull request:

- add capability to use the [getToken](https://docs.luigi-project.io/docs/luigi-client-api?section=gettoken) function inside the iframe based container 
- Sends a message to the container iframe based microfrontend to set the auth data. This allows LuigiClient to receive the token through the getToken function inside the microfrontend.
- Added Cypress tests
- Added unit tests

- **Potential future merge conflict with [Light DOM Container PR](https://github.com/SAP/luigi/pull/3697) since it affects LuigiContainer.svelte files** _due to the new Prettier for Svelte options recently added_ - merge that one first and resolve Merge Conflicts here afterward, to make reviewing this PR easier.

**Related issue(s)**

Resolves #3784

### CODE REVIEW

File: container/src/LuigiContainer.svelte [hunk 1]
```diff
@@ -6,74 +6,44 @@
...
         attributeChangedCallback(name, oldValue, newValue) {
-          if (this.containerInitialized && name === 'context') {
-            this.updateContext(JSON.parse(newValue));
+          if (this.containerInitialized) {
+            if (name === 'context') {
+              this.updateContext(JSON.parse(newValue));
+            }
+            if (name === 'auth-data') {
+              ContainerAPI.updateAuthData(this.iframeHandle, JSON.parse(newValue));
+            }
```
Comment (ndricimrr) [inline container/src/LuigiContainer.svelte:42]: Rest is prettier changes

File: container/src/LuigiCompoundContainer.svelte [hunk 2]
```diff
@@ -109,26 +109,20 @@
     const node = {
       compound: compoundConfig,
       viewUrl: viewurl,
-      webcomponent:
-        GenericHelperFunctions.checkWebcomponentValue(webcomponent) || true,
+      webcomponent: GenericHelperFunctions.checkWebcomponentValue(webcomponent) || true
     }; // TODO: fill with sth
-    webcomponentService
```
Comment (ndricimrr) [inline container/src/LuigiCompoundContainer.svelte:115]: whole file contains prettier changes only

**Comment by ndricimrr** [comment]
⚠️  Review after [Light DOM Container ](https://github.com/SAP/luigi/pull/3697)has been merged. To avoid reviewing prettifyed files.

File: container/test-app/iframe/iframeContainer.html [hunk 3]
```diff
@@ -161,15 +168,6 @@ <h3>
         );
       });
 
-      // SET_THIRD_PARTY_COOKIES_REQUEST - called on microfrontend startup
```
Comment (walmazacn) [inline container/test-app/iframe/iframeContainer.html:164]: Why did you remove this part? We decided to drop support for 3rd party cookies in Luigi v3, not in 'main' branch

File: container/test-app/wc/clientAPI.html [hunk 4]
```diff
@@ -37,6 +37,7 @@ <h3>
         viewURL="./helloWorldWCSelfRegistered.js"
         webcomponent='{"selfRegistered": "true"}'
     ></luigi-container> -->
+      <script></script>
```
Comment (walmazacn) [inline container/test-app/wc/clientAPI.html:40]: Why do we need this empty script tag?

File: container/test-app/iframe/iframeContainer.html (hunk 3 above)
Comment (ndricimrr) [reply container/test-app/iframe/iframeContainer.html]: true. reverted.

File: container/test-app/wc/clientAPI.html (hunk 4 above)
Comment (ndricimrr) [reply container/test-app/wc/clientAPI.html]: good catch. missed it. applied suggestions 👍

**Review by walmazacn — APPROVED** [review]
LGTM 👍
